MaxText/llama_or_mistral_ckpt.py

```diff
@@ -41,6 +41,7 @@
 from train import save_checkpoint
 import torch
 import sys
+import os
```

Comment:
remove

Reply:
we need this for enabling the device simulation flag
`os.environ['XLA_FLAGS'] = f'--xla_force_host_platform_device_count={SIMULATED_CPU_DEVICES_COUNT}'`